### PR TITLE
Make command lazy

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Fixed tickets | #27 
| License       | MIT

### CODE REVIEW

**Comment by kunicmarko20** [comment]
Not sure we can? we still support symfony 2.8? This is on `1.x`

File: src/Resources/config/command.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <services>
         <service id="fos_ck_editor.command.installer" class="FOS\CKEditorBundle\Command\CKEditorInstallerCommand">
             <argument type="service" id="fos_ck_editor.installer" />
-            <tag name="console.command" />
+            <tag name="console.command" command="ckeditor:install" />
```
Comment (ElectricMaxxx) [inline src/Resources/config/command.xml]: wouldn't that mean, we can omit the name definition in the class now?

Comment (kunicmarko20) [reply src/Resources/config/command.xml]: I am not sure, don't we still need it for sf 2.8?

Comment (kunicmarko20) [reply src/Resources/config/command.xml]: as any sf version < 3.4 doesn't know how to read `command="ckeditor:install" ` and will just ignore it.

Comment (kunicmarko20) [reply src/Resources/config/command.xml]: @ElectricMaxxx so?

**Pull request merged**